src/bitfield/mod.rs

```diff
@@ -57,6 +58,11 @@ impl Bitfield {
     }
   }
 
+  /// Get the current length
+  pub fn length(&self) -> usize {
```

Comment:
We should call this `fn len` to be consistent with all of stdlib's structures (:

Reply:
Good catch! It's fixed now.